### PR TITLE
Identify and access - Domain - Delete domain does not remove it from the tree

### PR DESCRIPTION
**Pull request by utchoang**
@rhtyd cc @svenvogel
Fixes https://github.com/apache/cloudstack-primate/issues/180. Please review and let me know your comment.

### CODE REVIEW

**Comment by svenvogel** [comment]
@utchoang cool thanks and good work. Yes we will test this.

**Comment by shwstppr** [comment]
@utchoang can you please check if this is working with the lastest master merge. For me, UI was still showing same deleted element details

**Comment by utchoang** [comment]
Hi @shwstppr. After a check, I found that the domain info still remains even though the domain is deleted. I fixed it, please check.

**Comment by shwstppr** [comment]
@utchoang after delete is it possible to move selection/highlight to deleted item's parent. Currently, it goes to root item

**Comment by shwstppr** [comment]
@rhtyd ready for review

File: src/config/section/storage.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default {
       permission: ['listVolumesMetrics', 'listVolumes'],
       resourceType: 'Volume',
       columns: ['name', 'state', 'type', 'vmname', 'size', 'physicalsize', 'utilization', 'diskkbsread', 'diskkbswrite', 'diskiopstotal', 'storage', 'account', 'zonename'],
-      details: ['name', 'id', 'type', 'storagetype', 'diskofferingdisplaytext', 'deviceid', 'sizegb', 'physicalsize', 'provisioningtype', 'utilization', 'diskkbsread', 'diskkbswrite', 'diskioread', 'diskiowrite', 'diskiopstotal', 'miniops', 'maxiops', 'path'],
```
Comment (rhtyd) [inline src/config/section/storage.js:30]: Fix this is reverting another fix.

Comment (utchoang) [reply src/config/section/storage.js]: I resolve it by merge from master.

**Comment by rhtyd** [comment]
@utchoang the code has created a lot of dependencies in AutogenView which it shouldn't - at some point in future technical debt needs to be paid by creating an entire new view component

**Pull request merged**